src/components/campaigns/grid/GridActions.tsx

```diff
@@ -9,11 +9,12 @@ import { routes } from 'common/routes'
 
 type Props = {
   id: string
+  allowDelete: boolean
   onView: () => void
   onDelete: () => void
 }
 
-export default function GridActions({ id, onView, onDelete }: Props) {
+export default function GridActions({ id, allowDelete, onView, onDelete }: Props) {
```

Comment:
@marinovl7 You can add a default value for `allowDelete` to avoid passing it to grids explicitly.

```tsx
type Props = {
  id: string
  allowDelete?: boolean
  onView: () => void
  onDelete: () => void
}

export default function GridActions({ id, allowDelete = false, onView, onDelete }: Props)
```